pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -47,7 +47,16 @@ export const CryptoPolicyRow = () => {
         cockpit.file("/proc/sys/crypto/fips_enabled").read()
                 .then(content => setFipsEnabled(content ? content.trim() === "1" : false));
         cockpit.file("/etc/crypto-policies/state/current")
-                .watch(content => setCurrentCryptoPolicy(content ? content.trim() : null));
+                .watch(async contents => {
+                    // Ask crypto-policies to get correct FIPS state, as that dominates the configured policy
+                    try {
+                        setCurrentCryptoPolicy((await cockpit.spawn(["update-crypto-policies", "--show"])).trim());
+                    } catch (error) {
+                        console.warn("Failed to get current crypto policy:", error.toString(),
+                                     "; falling back to /etc/crypto-policies/state/current");
+                        setCurrentCryptoPolicy(contents.trim());
+                    }
```

Comment:
Note: I don't know how this could happen, `--show` works unprivileged and really Should Not Fail™. This is just belt-and-suspenders, and can't be test-covered.